static/scss/homepage.scss

```diff
@@ -221,7 +221,9 @@ body.app-media {
       }
 
       @include breakpoint(phone) {
-        display: none;
+        font-size: 20px;
+        line-height: 1.15;
```

Comment:
Should `line-height` have an `em` unit? I notice a style a few lines above lacks it too, but that probably needs to get corrected too.